src/replay.ts

```diff
@@ -60,10 +61,12 @@ async function processMessages(messages: any[]) {
 
       if (message.type === 'delete-proposal') {
         console.log('New event: "delete-proposal"', message.space, message.id);
-        await handleDeletedEvent({
-          space: message.space,
-          ipfs: message.ipfs
-        });
+        await handleDeletedEvent(
+          {
+            space: message.space
+          },
+          message.ipfs
+        );
```

Comment:
Any reason? 😅

Reply:
ipfs is not part of the `Event` signature.

In the previous code:
`ipfs` was assigned to event, and deleted from it inside `handleDeletedEvent`, making the assignment to event just something to pass 1 argument to the function instead of 2.

In the current code:
Pass ipfs separately, and keep the event object clean.